cernopendata/modules/fixtures/data/records/jade-computing-notes.json

```diff
@@ -0,0 +1,5267 @@
+[
+  {
+    "abstract": {
+      "description": "Index"
+    },
+    "authors": "",
+    "accelerator": "DESY-PETRA",
+    "collaboration": {
+      "name": "JADE collaboration",
+      "recid": "451"
+    },
+    "collections": [
+      "JADE-Computing-Notes"
+    ],
+    "date_created": [
```

Comment:
"date_created" is mandatory and cannot be empty. If you don't know the year, perhaps we should use some placeholder such as 1970 or anything you wish?